### PR TITLE
Added Arabic language

### PR DESCRIPTION
**Pull request by mrkhalidj**
- translated the application into Arabic.
- added Arabic language to the drop-down list so users can choose it.
- added Arabic language to locales.

hope this will help more people to use this application 😃

### CODE REVIEW

**Comment by ssddanbrown** [comment]
Thank you @kmoj86 for the work you put into this. A couple of questions:

1. Does everything generally work okay? From my understanding Arabic is a right-to-left language so I would have expected more work to be required to support such a language.
2. Is it okay if I squash this before merging? Just to reduce the amount of commits that will be merged for a cleaner history. Will still show as yourself being the author.

**Comment by kmoj86** [comment]
@ssddanbrown 

hello dan,

- yes, everything works ok. but it's not the ideal way to view the application in Arabic. for best results, the CSS and text editor need to be configured for RTL. CSS only needs to be mirrored for the most part, the text editor on the other hand has to be reconfigured to handle RTL languages (Froala's WYSIWYG editor as an example).
But nothing that renders the app unusable. I've already tested it and the found no issues with it.
That being said, this is a step forward in supporting Arabic language by translating the user interface, so anyone who doesn't speak a second language can use the application. and if one day a more experienced contributor comes to add RTL support for a better user experience, then the translation will already be here for them to use.
**In short: yes, everything is _mostly_ ok** 😄 

- sure thing! you can squash this for a cleaner history.
this is my first pull request and I've been working on the translation on 2 different PCs, so sorry for the mess.

**Comment by ssddanbrown** [comment]
Thanks again for this, Will be in the next release. My notes on RTL support here: https://github.com/BookStackApp/BookStack/issues/939#issuecomment-423740828

Didn't squash the commits in the end (Didn't know how to git-squash properly with an active merge conflict :laughing: )

**Comment by kmoj86** [comment]
@ssddanbrown 
Great! thanks for all your work!
I tried to add rtl support myself by adding directionality plugin for TinyMCE, but for some reason it just didn't work for me. so this is really great news!
